doc/source/api.rst

```diff
@@ -6,19 +6,18 @@ API Reference
 *************
 
 This page gives an overview of all public pandas objects, functions and
-methods. In general, all classes and functions exposed in the top-level
-``pandas.*`` namespace are regarded as public.
+methods. All classes and functions exposed in ``pandas.*`` namespace are public.
 
-Further some of the subpackages are public, including ``pandas.errors``,
-``pandas.plotting``, and ``pandas.testing``. Certain functions in the
-``pandas.io`` and ``pandas.tseries`` submodules are public as well (those
-mentioned in the documentation). Further, the ``pandas.api.types`` subpackage
-holds some public functions related to data types in pandas.
+Some subpackages are public which include ``pandas.errors``,
```

Comment:
these all could be references

Reply:
Hello @jreback , sorry, I don't understand your comment. Could you clarify what are the further changes that I need to address for this commit? Thanks a lot.

Reply:
use ``:module:`pandas.errors` `` and so on

Reply:
just uploaded a new fix. Please let me know if I am still doing it wrong. Thanks

Reply:
@jreback we don't have module docstring pages, so I don't think such links will work

Reply:
Hi @jorisvandenbossche , thanks for the clarification. Just revert the commit back to origin: dccfee5